core/modules/simpletest/tests/upgrade/upgrade.test

```diff
@@ -207,7 +207,15 @@ abstract class UpgradePathTestCase extends BackdropWebTestCase {
     // Check if there still are pending updates.
     $this->backdropGet($update_url, array('external' => TRUE));
     $this->backdropPost(NULL, array(), t('Continue'));
-    if (!$this->assertText(t('No pending updates.'), 'No pending updates at the end of the update process.')) {
+
+    // Install new updates if any.
+    if ($this->assertText(t('Updates have been found that need to be applied'))) {
+      $this->backdropPost(NULL, array(), t('Apply pending updates'));
+      if (!$this->assertText(t('Updates were attempted'))) {
+        return FALSE;
+      }
+    }
+    elseif (!$this->assertText(t('No pending updates.'), 'No pending updates at the end of the update process.')) {
```

Comment:
>  So when layout installs with my new layout_update_1000(), performUpgrade() finds this and terminates the upgrade. This little code block anticipates this.

Ah right. So this code block adjusts how the upgrade process works (both for tests and end-users). This means that you'd need to run update.php twice in order to upgrade a site from D7 to Backdrop. In other situations where we've needed to run updates to modules that did not exist in Drupal 7 but are enabled by default in Backdrop, we put the updates in system.install. I think that would be appropriate for this situation as well, instead of making `layout_update_1000()`.